server/backend/api/app/routers/router_gc_notify.py

```diff
@@ -0,0 +1,17 @@
+import logging
+
+from api.app.integration.gc_notify import GCNotifyEmailService
+from api.app.schemas import GCNotifyEmailParam
+from fastapi import APIRouter
+
+LOGGER = logging.getLogger(__name__)
+
+router = APIRouter()
+
+
+@router.post("/send", response_model=str)
```

Comment:
In terms of the design, its better security not to expose the actual send_email endpoint, and just have other back-end processes generate (send) emails when required.

Reply:
I could remove it for now, and we could just call the curd method when need to send the email. I believe that will in the process of create user role assignment.